cmd/utils/flags.go

```diff
@@ -812,7 +812,7 @@ func GetHttpPort(nodeLocation common.Location) int {
 	case common.REGION_CTX:
 		return 9002 + nodeLocation.Region()
 	case common.ZONE_CTX:
-		return 9100 + 20*nodeLocation.Region() + nodeLocation.Zone()
+		return 9200 + 20*nodeLocation.Region() + nodeLocation.Zone()
```

Comment:
lets maybe then change the websocket to 8200 as well, its easier to think that way